linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureAstTransformer.scala

```diff
@@ -322,7 +322,9 @@ private class ClosureAstTransformer(featureSet: FeatureSet,
         args.foreach(arg => node.addChildToBack(transformExpr(arg)))
         node
       case DotSelect(qualifier, item) =>
-        new Node(Token.GETPROP, transformExpr(qualifier), transformString(item))
+        val node = Node.newString(Token.GETPROP, item.name)
+        node.addChildToBack(transformExpr(qualifier))
+        node
```

Comment:
Should we give this node the position of the ident or is the position of the select fine?

Reply:
I think it's fine as is. If we want to give it the position of the ident, we'll have to somehow refactor the code in a non obvious way.

Reply:
> we'll have to somehow refactor the code in a non obvious way.

That is the lamest excuse I've heard in a while :P 

More seriously: Do we have any idea how this affects source maps? I'd expect not really, because the position of the ident and the select probably just differ by 1 column (dot v.s. what's after the dot). So yes, let's just go with this.

Reply:
> That is the lamest excuse I've heard in a while :P

Yes :P But it would be surprisingly annoying to change this. Currently we have a global assumption that the position of the GCC Node is the position of the encompassing `js.Tree`.

> More seriously: Do we have any idea how this affects source maps? I'd expect not really, because the position of the ident and the select probably just differ by 1 column (dot v.s. what's after the dot). So yes, let's just go with this.

We never know how source maps are affected. They're this weird blackbox thing.

The best we can do, I think, is mimic what GCC does for its own nodes. It seems like it uses the position of the identifier at
https://github.com/google/closure-compiler/blob/f25ce466cb6610affa73d6514e9ba70f2f0e4c46/src/com/google/javascript/jscomp/parsing/IRFactory.java#L2054
so I'll try a bit harder to do that annoying refactoring.